packages/lit-html/src/directives/cache.ts

```diff
@@ -94,6 +95,7 @@ export const cache = directive(
           // Move cached part back into DOM
           clearPart(containerPart);
           insertPart(containerPart, undefined, cachedPart);
+          cachedPart.setConnected(true);
```

Comment:
Does it matter if this is after or before the next line putting the `cachedPart` into the `committedValue` of the `containerPart`?

Reply:
Hmm, yeah it might be better to do it after, since it runs user code...? Will make that change.

Reply:
Done.